cudamapper/CMakeLists.txt

```diff
@@ -13,6 +13,22 @@ project(cudamapper)
 # Process data subdirectory first
 add_subdirectory(data)
 
+execute_process(COMMAND
+  git describe --tag --dirty
+  WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
+  OUTPUT_VARIABLE VERSION
+  OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+string(REGEX REPLACE "^v([0-9]+)\\..*" "\\1" VERSION_MAJOR "${VERSION}")
+string(REGEX REPLACE "^v[0-9]+\\.([0-9]+).*" "\\1" VERSION_MINOR "${VERSION}")
+string(REGEX REPLACE "^v[0-9]+\\.[0-9]+\\.([0-9]+).*" "\\1" VERSION_PATCH "${VERSION}")
+string(REGEX REPLACE "^v[0-9]+\\.[0-9]+\\.[0-9]-+(.*)" "\\1" VERSION_SHA1 "${VERSION}")
+set(VERSION_SHORT "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/templates/version.cpp.in
```

Comment:
I think this file should belong to the `claragenomics/common/base/include/claragenomics` folder. Either added to cga_config.hpp or a new version.hpp. What do you think @ahehn-nv ?

Reply:
A separate `version.hpp` is fine I think.